apps/crn-frontend/src/discover/tutorials/__tests__/Tutorial.test.tsx

```diff
@@ -0,0 +1,80 @@
+import { RecoilRoot } from 'recoil';
+import { Suspense } from 'react';
+import { render, waitFor } from '@testing-library/react';
+import { MemoryRouter, Route } from 'react-router-dom';
+import { TutorialsResponse } from '@asap-hub/model';
+import { tutorials } from '@asap-hub/routing';
+
+import Tutorial from '../Tutorial';
+
+import { Auth0Provider, WhenReady } from '../../../auth/test-utils';
+import { refreshTutorialItemState } from '../state';
+import { getTutorialById } from '../api';
+
+jest.mock('../api');
+
+const tutorial: TutorialsResponse = {
+  id: '55724942-3408-4ad6-9a73-14b92226ffb6',
+  created: '2020-09-07T17:36:54Z',
+  title: 'Tutorial Title',
+};
+
+const mockGetTutorialById = getTutorialById as jest.MockedFunction<
+  typeof getTutorialById
+>;
+
+const renderPage = async () => {
+  const result = render(
+    <RecoilRoot
+      initializeState={({ set }) =>
+        set(refreshTutorialItemState(tutorial.id), Math.random())
+      }
+    >
+      <Suspense fallback="loading">
+        <Auth0Provider user={{}}>
+          <WhenReady>
+            <MemoryRouter
```

Comment:
I am not sure why you are mocking the router when Tutorial doesn't have any router logic